AmplifyPlugins/Core/AWSPluginsCore/Keychain/KeychainStoreAttributes.swift

```diff
@@ -22,6 +22,7 @@ extension KeychainStoreAttributes {
             KeychainStore.Constants.Class: itemClass,
             KeychainStore.Constants.AttributeService: service
         ]
+        query[KeychainStore.Constants.UseDataProtectionKeyChain] = kCFBooleanTrue
```

Comment:
Why not add it to the inline array above?

Also if the `UseDataProtectionKeyChain` key is now here, then adding it again in `defaultSetQuery()` seems redundant.

Reply:
Updated per suggestion